src/types.db

```diff
@@ -192,6 +192,27 @@ snr			value:GAUGE:0:U
 spam_check		value:GAUGE:0:U
 spam_score		value:GAUGE:U:U
 spl			value:GAUGE:U:U
+stenographer_aged_files value:COUNTER:0:U 
+stenographer_current_files value:COUNTER:0:U
+stenographer_http_request_query_POST_bytes value:COUNTER:0:U
+stenographer_http_request_query_POST_completed value:COUNTER:0:U
+stenographer_http_request_query_POST_nanos value:COUNTER:0:U
+stenographer_index_base_lookup_nanos value:COUNTER:0:U
+stenographer_index_base_lookups_finished value:COUNTER:0:U
```

Comment:
For `index_{base,set}_lookups_{finished,started}` I would use the existing `total_operations` type.